test/test_oneagentmetadataenricher.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import unittest
+from unittest.mock import patch
+
+from dynatrace.opentelemetry.metrics.export import OneAgentMetadataEnricher
+
+
+class TestOneAgentMetadataEnricher(unittest.TestCase):
```

Comment:
Please add tests for the file handling (redirection file not being present or empty, containing nothing or an invalid file name, actual file not being present or empty, containing invalid or empty data).

Reply:
added a bunch of tests